draft-irtf-panrg-path-properties.md

```diff
@@ -272,7 +272,7 @@ Symmetric Path:
 For example, a path which consists of layer 3 switches and links between them and a reverse path with the same path elements but in reverse order are considered "routing" symmetric, as the same path elements on the same level of abstraction (IP forwarding) are traversed in the opposite direction.
 
 Path MTU:
-: The maximum size, in octets, of an IP packet that can be transmitted without fragmentation.
+: The maximum size, in octets, of a packet that can be transmitted without fragmentation.
```

Comment:
```suggestion
: The maximum size, in octets, of a packet (or frame) that can be transmitted without fragmentation.
```

Reply:
Thanks for the suggestion, I'll use it but without the brackets :slightly_smiling_face: